### PR TITLE
ManualPayStation solution changes to support common architecture

### PR DESCRIPTION
**Pull request by ankitit2015**

### CODE REVIEW

File: Parking.Common/ConfigurationReader.cs [hunk 1]
```diff
@@ -9,14 +10,16 @@ public sealed class ConfigurationReader
         private static readonly string ConfigFilePath = Path.Combine(Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().GetName().CodeBase.Substring(8)), configurationFileName);
 
         //TD Specific
```
Comment (mohitrawat789) [inline Parking.Common/ConfigurationReader.cs]: Please remove the comment as now its not TD specific

File: Parking.Common/ISqlDataAccess.cs [hunk 2]
```diff
@@ -64,7 +64,6 @@ public interface ISqlDataAccess
         /// <summary>
         ///
```
Comment (mohitrawat789) [inline Parking.Common/ISqlDataAccess.cs]: Please Remove empty comment

File: Parking.Common/Model/ManualPayStationSettings.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿namespace Parking.Common
+{
+    public class ManualPayStationSettings
+    {
+        public string DeviceId { get; set; }
+        public string UserId { get; set; }
+        public string Password { get; set; }
+        public string TdServerIPAddress { get; set; }
+        public string TdServerPort { get; set; }
```
Comment (mohitrawat789) [inline Parking.Common/Model/ManualPayStationSettings.cs]: TdServerPort as string? i guess it should be int

File: Parking.Common/Model/ManualPayStationSettings.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        public string TdServerUsername { get; set; }
+        public string TdServerPassword { get; set; }
+        public string VehicleStatusPassword { get; set; }
+        public string FourWheelerParkingSpace { get; set; }
```
Comment (mohitrawat789) [inline Parking.Common/Model/ManualPayStationSettings.cs]: Should be Int

File: Parking.Common/Model/ManualPayStationSettings.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        public string TwoWheelerParkingSpace { get; set; }
```
Comment (mohitrawat789) [inline Parking.Common/Model/ManualPayStationSettings.cs]: Should be Int

File: Parking.Common/Model/ManualPayStationSettings.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        public string FourWheelerParkingRate { get; set; }
+        public string TwoWheelerParkingRate { get; set; }
```
Comment (mohitrawat789) [inline Parking.Common/Model/ManualPayStationSettings.cs]: Should be float or decimal

File: Parking.Common/ConfigurationReader.cs (hunk 1 above)
Comment (ankitit2015) [reply Parking.Common/ConfigurationReader.cs]: Done

File: Parking.Common/ISqlDataAccess.cs (hunk 2 above)
Comment (ankitit2015) [reply Parking.Common/ISqlDataAccess.cs]: Done

File: Parking.Common/Model/ManualPayStationSettings.cs (hunk 3 above)
Comment (ankitit2015) [reply Parking.Common/Model/ManualPayStationSettings.cs]: Done

File: Parking.Common/Model/ManualPayStationSettings.cs (hunk 5 above)
Comment (ankitit2015) [reply Parking.Common/Model/ManualPayStationSettings.cs]: Done

File: Parking.Common/Model/ManualPayStationSettings.cs [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+        public string UserId { get; set; }
+        public string Password { get; set; }
+        public string TdServerIPAddress { get; set; }
+        public string TdServerPort { get; set; }
+        public string TdServerUsername { get; set; }
+        public string TdServerPassword { get; set; }
+        public string VehicleStatusPassword { get; set; }
+        public string FourWheelerParkingSpace { get; set; }
+        public string TwoWheelerParkingSpace { get; set; }
+        public string FourWheelerParkingRate { get; set; }
```
[the remark this answers is not in the document]
Comment (ankitit2015) [reply Parking.Common/Model/ManualPayStationSettings.cs]: Done

File: Parking.Common/Model/ManualPayStationSettings.cs (hunk 6 above)
Comment (ankitit2015) [reply Parking.Common/Model/ManualPayStationSettings.cs]: Done

File: Parking.Common/Model/ManualPayStationSettings.cs [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+        public string TwoWheelerParkingRate { get; set; }
+        public string LostTicketPenality { get; set; }
+    }
```
[the remark this answers is not in the document]
Comment (ankitit2015) [reply Parking.Common/Model/ManualPayStationSettings.cs]: Done

File: Parking.Common/Model/TickerDispenserClientSettings.cs [hunk 9]
```diff
@@ -22,5 +22,7 @@ public class TDClientSetting
         public string TdServerPort { get; set; }
         public string TdServerUsername { get; set; }
         public string TdServerPassword { get; set; }
+        public string FourWheelerParkingSpace { get; set; }
```
[the remark this answers is not in the document]
Comment (ankitit2015) [reply Parking.Common/Model/TickerDispenserClientSettings.cs]: Let it be here, Future prospective

File: Parking.Common/Model/TicketDispenserServerSettings.cs [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Parking.Common
+{
+    public class TicketDispenserServerSettings
+    {
+        public string IPAddress { get; set; }
+        public string Port { get; set; }
```
[the remark this answers is not in the document]
Comment (ankitit2015) [reply Parking.Common/Model/TicketDispenserServerSettings.cs]: Done

**Comment by ankitit2015** [comment]
Merged

**Pull request merged**